Source/Tools/run_diff.py

```diff
@@ -109,7 +109,7 @@ def get_changed_labview_file_paths(repo_root_directory, pr_number, token):
     else:
         changed_files = get_git_diff_changed_files()
 
-    diff_regex = re.compile(r"^(.*\.vi[tm]?)|(.*\.ctl?)$", re.MULTILINE)
+    diff_regex = re.compile(r"^.*\.(vi|vit|vim|ctl)$", re.MULTILINE)
```

Comment:
I updated this regex because the old match was considering VI Analyzer config files (`.viancfg`) a match, thus would've otherwise gone down the path of producing diff imagery.